lib/core.yaml

```diff
@@ -0,0 +1,28 @@
+linter:
```

Comment:
I wonder if we want the file path here to be `/lib/dart/core.yaml` so that we could also have a `/lib/flutter/core.yaml` ?

Reply:
Interesting.  Scoping has some other nice benefits too.

@goderbauer @devoncarew : wdyt?

Reply:
TBH I think that supporting frameworks in this package will be challenging. Once we add one there's no reason we shouldn't then support more; changes here will be tightly versioned together even though the frameworks will be logically unrelated.

I'd optimize for the 'dart' case here, and just have the two main files in the top level of the `lib/` directory.

Reply:
> I'd optimize for the 'dart' case here,

I guess if we leave it as is, we could always add directories for frameworks (if we added them).  So we might have something like:

    lib/
     core.yaml
     recommend.yaml
     framework_1/
       core.yaml
       recommend.yaml
     framework_2/
       core.yaml
     ...


What do we think about `flutter` in the short term?  Top-level, nested or somewhere else?

Reply:
FWIW, I don't think we will have multiple lint sets for flutter. There'll just be a `flutter.yaml` set,